graph/path/bellman_ford_moore.go

```diff
@@ -27,10 +30,63 @@ func BellmanFordFrom(u graph.Node, g graph.Graph) (path Shortest, ok bool) {
 	path = newShortestFrom(u, nodes)
 	path.dist[path.indexOf[u.ID()]] = 0
 
-	// TODO(kortschak): Consider adding further optimisations
-	// from http://arxiv.org/abs/1111.5414.
```

Comment:
This should not be removed. The optimisation you are adding is one, there are others discussed there.